src/cache/types.ts

```diff
@@ -1,8 +1,8 @@
 export interface IIORedisCacheOptions {
   host: string;
   password: string;
-  port: number;
```

Comment:
are you sure this is optional?
from the name of the branch/ticket it seems like the required fields should be host & port

Reply:
changed